cmd/build/v2/build_test.go

```diff
@@ -367,3 +347,74 @@ func createDockerfile(t *testing.T) (string, error) {
 	}
 	return dir, nil
 }
+
+func TestBuildWithDependsOn(t *testing.T) {
```

Comment:
How are we testing that a has been built before b?

Reply:
We check that the image `a` has the image environment variables of `b`